src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -539,7 +539,9 @@ class RsCompletionTest : RsCompletionTestBase() {
     """)
 
     fun `test hidden macro is hidden in other module`() = checkNoCompletion("""
+        #[doc="No problems with"]
         #[doc(hidden)]
+        #[doc="explicit docs"]
```

Comment:
Let's create separate test with several `#[doc]` attributes

Reply:
Done, i just created a separate test for this instead of modifying the existing test.